src/lib.rs

```diff
@@ -75,13 +87,15 @@ fn camera_movement(
 pub struct PanCam {
     pub grab_buttons: Vec<MouseButton>,
     pub enabled: bool,
+    pub track_mouse: bool,
```

Comment:
I wonder if we can find a better name for this? zoom_to_pointer? cursor/mouse?

Currently, it's kind of hard to guess what it is from only the name.

Reply:
Sure

Reply:
zoom_to_cursor sounds like a better name